npm/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@tailcallhq/tailcall",
+  "version": "0.13.0",
+  "type": "module",
+  "files": [
+    "src/*",
+    "target/*"
+  ],
+  "description": "A high-performance GraphQL proxy, optimized for cloud-native environments, covering edge, middle, and service layers.",
+  "main": "./src/index.js",
+  "bin": "./src/index.js",
```

Comment:
Are these required?

Reply:
Yeah, won't work without it, npm packages need an entry point.

Reply:
it's possible to specify shell script or shell command here instead of js file that won't spin up the nodejs runtime to run the binary